pkg/cmd/pr/shared/finder.go

```diff
@@ -154,6 +154,12 @@ func (f *finder) Find(opts FindOptions) (*api.PullRequest, ghrepo.Interface, err
 		}
 	}
 
+	if fields.Contains("projectItems") {
+		if !api.HasProjectsV2Scope(httpClient, f.repo.RepoHost()) {
```

Comment:
It will return false and then skip the projectsV2 look up. The classic projects still get looked up and work just as before.

What sort of graceful token permission error handling are you envisioning? I took the approach, throughout this PR, that anytime projectsV2 are not supported (due to scopes or due to GHES version) that it the command would continue on functioning as it did previously and would not alert the user. This felt like the least intrusive and least workflow breaking approach for our users.

Reply:
> Do App Tokens and PAT v2 tokens even have access to the graphql API?

App Tokens do have access to projects via the `repository-projects` permission. You are right that PAT v2 can not access the GraphQL API, but that will change in the future. When that future comes, it would be good if gh isn't unnecessarily tied to scopes-based tokens.

> I took the approach, throughout this PR, that anytime projectsV2 are not supported (due to scopes or due to GHES version) that it the command would continue on functioning as it did previously and would not alert the user.

I appreciate that you took that approach! What I'm proposing is that we continue with the same approach, but also try to query `projectItems` while being agnostic to the type of access token. That could be done by making the `projectItems` query separately (without checking token scopes) and just ignore permissions errors in the response.

Reply:
Ah I understand. That makes sense to me. I will make that change.